src/Pusher.php

```diff
@@ -158,6 +169,7 @@ public function set_logger($logger)
     private function log($msg)
     {
         if (is_null($this->logger) === false) {
+            /* @noinspection PhpUndefinedMethodInspection */
```

Comment:
Is this a phpstorm-specific annotation? I'm apprehensive about putting IDE-specific annotations in here. Is there a way we can fix the underlying problem so that this does not need to be suppressed at all? Also, could you link to the docs on this annotation? I can't find much about it online...

Reply:
Good catch, I will remove it and add following in the top, it is enough to remove the warning (at the moment)
```
 /**
   * @var null | object
   */
private $logger = null;
```

Reply:
It looks like you haven't made the change you described yet..